index.js

```diff
@@ -72,6 +72,8 @@ app.use(
     const accessToken = request.headers["x-access-token"]
     const userID = request.headers["x-user-id"]
     const timezone = request.headers["x-timezone"]
+    const requestID = request.headers["x-request-id"] || "implement-me"
```

Comment:
you're going to get _a lot_ of this to start with, perhaps you could change it to be the req user agent?

Reply:
I think this is okay for now because heroku generates an `x-request-id` for each request (unless one is specified, like from force). But I can revisit later if it is an issue.